lib/ovirt/vm.rb

```diff
@@ -367,8 +367,7 @@ def converted_cloud_init(content)
       require 'yaml'
       raw_content          = YAML.load(content)
       hash                 = ovirt_cloud_init_keys.each_with_object({}) { |k, h| h[k] = raw_content.delete(k) }
-      custom_script        = YAML.dump(raw_content).to_s.sub("---\n", "")
-      hash[:custom_script] = custom_script unless custom_script.blank?
+      hash[:custom_script] = YAML.dump(raw_content).to_s.sub("---\n", "") if raw_content.present?
```

Comment:
`YAML.dump` is always returning a string so the `.to_s` call is not needed.  Can you give a little detail behind striping the YAML header here?